### ISSUE TITLE
ctrl + click to a function shows the interface , than the actual implementation

### ISSUE THREAD

**Issue by JADcooler** [NomicFoundation/hardhat-vscode#538]
feature request where a shortcut, like ctrl + shift + a, can be used to get the implementation of a function in an interface.

**Comment by kanej** [comment]
Is the built in support for jumping to implementations enough to cover this request:

![image](https://github.com/user-attachments/assets/0e055e55-e60c-47c0-a236-80d50738cea9)

**Comment by JADcooler** [comment]
no, it works only for the source file. Go to implementation doesn't work when done for the interface file.
Only find all references works for that, ctrl + click doesn't
